get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -97,174 +147,111 @@ m|:help keys
+You can also turn previously dragged nodes loose again.
 
-If you enter fullscreen mode of a graph visualization you can zoom in and out.
-After a node is clicked it gets a halo, in which you can expand and remove nodes from the visualization. 
-You can also turn previously dragged nodes lose again.
+== Graph Metadata
 
-== Meta Graph
+In the left-side menu, the top icon is the database section (icon:database[]), where you can find the *currently used* node labels, relationship types, and property keys.
+Clicking on any one of those options runs a quick query to show you a sample of the graph with those elements.
 
-In the left side drawer on the top database section (icon:database[]) you find the *currently used* node-labels and relationship types.
-Clicking on any of those runs a quick query to show you a sample of the graph using those.
+== Setting Favorites
 
-== Queries and Favorites
+If you currently have an empty frame, you can display some nodes and relationships by using the Favorites (icon:star-o[]) in the left-side menu.
+Neo4j stores a few default favorites to get you started.
+Just click on the *Basic Queries*, then choose **Get Some Data** and run the query.
+This executes the statement `MATCH (n) RETURN n limit 100`, which fetches some nodes.
 
-If you start with an empty frame, display some nodes and relationships, use the Favorites (icon:star-o[]) drawer on the left, click on the **Get Some Data** entry, and run the query.
-This executes the statement `MATCH (n) RETURN n limit 100` which fetches some nodes. 
+You can save your own queries as favorites by "starring" them.
+Just populate the Browser command line with the query you want to favorite, then click the Favorites (icon:star-o[]) icon to the right of the command line.
+This will add the query to your Favorites list in the left-side menu.
+To run one of your Favorites, click on the left-side menu Favorites, choose the query, and run it.
 
-The browser helpfully also fetches and displays relationships between those nodes, *even if they were not part of your query result*.
-You can disable the latter behavior with the "Auto-Complete" switch in the bottom right corner.
-Then only relationships returned by the actual query will be shown.
+To provide a title or helpful info, you can use a comment `// comment` above your query to provide a title.
 
-You can save your own queries as favorites by "starring" them. 
-Use a comment `// comment` above your query to provide a title.
-Use folders to organize the favorites you can rearrange them by dragging and delete if they are no longer useful.
+Creating folders can help organize your favorites, and you can rearrange them by dragging or delete them if they are no longer useful.
 
 [NOTE]
 Your favorites, settings, and styles can be saved to the cloud with Neo4j Browser Sync.
 If you clear your browser cache, your local favorites will be removed.
 
-=== Advanced Styling 
```

Comment:
it would be cool if you recorded a screencast at some point :)

Reply:
Agree! Thought I'd get this started first, then add a bunch of screencasts to several pages later. :)